x-pack/plugins/security_solution/public/index.ts

```diff
@@ -9,6 +9,7 @@ import type { PluginInitializerContext } from '@kbn/core/public';
 import { Plugin } from './plugin';
 import type { PluginSetup, PluginStart } from './types';
 export type { TimelineModel } from './timelines/store/timeline/model';
+export { emptyLastBreadcrumbUrl } from './common/components/navigation/helpers';
```

Comment:
I am not sure this is a good idea, we should only export types and the plugin here. Adding direct exports like this here can make our bundle size increase very quickly.

It is starting to make sense to create a package with navigation-related stuff, only for Security, including helpers like this one. By now I think it is better to just implement it again in ess_security